src/pybind/ceph_argparse.py

```diff
@@ -851,30 +853,20 @@ def matchnum(args, signature, partial=False):
     return matchcnt
 
 
-def get_next_arg(desc, args):
+def _get_next_arg(desc, args):
     '''
     Get either the value matching key 'desc.name' or the next arg in
```

Comment:
I guess this comment changes with the reduced function too

Reply:
Yeah -- I think actually I'll remove this function entirely, the part where it considers one of the args being a list seems to be redundant as we're always parsing a simple list of strings from the CLI.